idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -426,22 +427,22 @@ def rule_cp_cold(blk, t):
                           self.PH, initialize=390, units=pyunits.K,
                           doc="Cold Temperature OUT of pass")
 
-        # ======================================================================
+        # =====================================================================
         # PERFORMANCE EQUATIONS
         # mass flow rate in kg/s
         def rule_mh_in(blk, t):
-            return blk.hot_side.properties_in[t].flow_mol *\
-                blk.hot_side.properties_in[t].mw
+            return blk.hot_fluid.properties_in[t].flow_mol *\
+                blk.hot_fluid.properties_in[t].mw
         self.mh_in = Expression(self.flowsheet().time, rule=rule_mh_in,
                                 doc='Hotside mass flow rate [kg/s]')
 
         def rule_mc_in(blk, t):
-            return blk.cold_side.properties_in[t].flow_mol *\
-                blk.cold_side.properties_in[t].mw
+            return blk.cold_fluid.properties_in[t].flow_mol *\
+                blk.cold_fluid.properties_in[t].mw
         self.mc_in = Expression(self.flowsheet().time, rule=rule_mc_in,
                                 doc='Coldside mass flow rate [kg/s]')
 
-        # ----------------------------------------------------------------------
+        # ---------------------------------------------------------------------
         # port mass velocity[kg/m2.s]
         def rule_Gph(blk, t):
             return (4 * blk.mh_in[t] * 7) / (22 * blk.port_dia**2)
```

Comment:
Where do these hard coded constants come from? There should at a minimum be a comment here with an explanation or reference for the source of these.